tests/kuttl/tests/deployments/02-assert-conductor-groups.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: ironic.openstack.org/v1beta1
+kind: Ironic
+metadata:
+  name: ironic
+  namespace: openstack
+spec:
+  serviceUser: ironic
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  storageClass: local-storage
```

Comment:
Will local-storage work when these tests run in CI?

Reply:
@cescgina ^

Reply:
@jparoly I believe we've typically left similar `storageClass` comparisons out of KUTTL asserts due to the fact that they might be different from one environment to another